sentry_sdk/integrations/django/signals_handlers.py

```diff
@@ -21,9 +21,14 @@ def patch_signals():
 
     def _get_receiver_name(receiver):
         # type: (Callable[..., Any]) -> str
-        name = receiver.__module__ + "."
+        if hasattr(receiver, "__module__"):
```

Comment:
can't we just use [`__qualname__`](https://docs.python.org/3/glossary.html#term-qualified-name) here?

Reply:
Good idea.

But I checked, the partials also do not have a `__qualname__`, so: no.

Reply:
oh, but we can use `__qualname__` if there is one, and if not we use `__name__`